source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/FactSetRenderer.java

```diff
@@ -82,8 +82,16 @@ public View render(
 
         FactVector factVector = factSet.GetFacts();
         long factVectorSize = factVector.size();
+
+        if (factVectorSize == 0)
```

Comment:
This seems like an extra validation that doesn't yield a lot of optimization, we could do it without this validation and just deal with the one below